config/filter.d/zoneminder.conf

```diff
@@ -0,0 +1,23 @@
+# Fail2Ban filter for Zoneminder login failures
+#
+
+[Definition]
+
+# pattern: [client 10.1.1.1:38022] WAR [Login denied for user "test"], referer: https://zoneminderurl/
+#
+#
+# Option:  failregex
+# Notes.:  regex to match the password failure messages in the logfile. The
+#          host must be matched by a group named "host". The tag "<HOST>" can
+#          be used for standard IP/hostname matching and is only an alias for
+#          (?:::f{4,6}:)?(?P<host>[\w\-.^_]+)
+
+
+failregex = ^.*? [[]client <HOST>:\d\d\d\d\d] WAR [[]Login denied for user \S*], referer: \S*$
```

Comment:
and paranoid me loves avoiding '.*' greedy or not.  What about just replacing it with smth like
`^\[\S*\] (?:[:error] )?(?:[pid \d*] )?`
(initial beast is probably needed to absorb the beginning of the line with date stripped out)